test/runtime/ct.go

```diff
@@ -49,15 +52,18 @@ func (c connTest) String() string {
 
 var _ = Describe("RuntimeConntrackTable", func() {
```

Comment:
Since the old CT test is now deprecated, this test should have its `Describe` changed from `RuntimeConntrackTable` --> `RuntimeValidatedConntrackTable` so that is run on `Cilium-Ginkgo-Tests` for PR builds as well as `Cilium-Ginkgo-Tests-All` which runs all tests.

Reply:
Done